concepticondata/conceptlists/Sidwell-2021-100.tsv

```diff
@@ -0,0 +1,101 @@
+ID	NUMBER	ENGLISH	SWADESH_1952_200	SWADESH_1955_100	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Sidwell-2021-100-1	1	Belly/stomach	10	4	2114	BELLY OR STOMACH
+Sidwell-2021-100-2	2	Big/large	11	5	1202	BIG
+Sidwell-2021-100-3	3	Bird	12	6	937	BIRD
+Sidwell-2021-100-4	4	Bite(v.)		7	1403	BITE
+Sidwell-2021-100-5	5	Black	14	8	163	BLACK
+Sidwell-2021-100-6	6	Blood	15	9	946	BLOOD
+Sidwell-2021-100-7	7	Bone		10	1394	BONE
+Sidwell-2021-100-8	8	Breast		11	1402	BREAST
+Sidwell-2021-100-9	9	Burn(intr.)		12	2102	BURN
+Sidwell-2021-100-10	10	Claw/nail		13	2128	CLAW OR NAIL
+Sidwell-2021-100-11	11	Cloud		14	1489	CLOUD
+Sidwell-2021-100-12	12	Cold	22	15	1287	COLD
+Sidwell-2021-100-13	13	Come	23	16	1446	COME
+Sidwell-2021-100-14	14	Dead	27	17	1625	DEAD
+Sidwell-2021-100-15	15	Dog	30	18	2009	DOG
+Sidwell-2021-100-16	16	Drink(v.)	31	19	1401	DRINK
+Sidwell-2021-100-17	17	Dry(stat.)		20	1398	DRY
+Sidwell-2021-100-18	18	Ear	35	21	1247	EAR
+Sidwell-2021-100-19	19	Earth/soil		22	1228	EARTH (SOIL)
+Sidwell-2021-100-20	20	Eat(any food)	37	23	1336	EAT
+Sidwell-2021-100-21	21	Egg	39	24	744	EGG
+Sidwell-2021-100-22	22	Eye	39	25	1248	EYE
+Sidwell-2021-100-23	23	Fire	48	27	221	FIRE
+Sidwell-2021-100-24	24	Flesh/meat	96	28	2615	FLESH OR MEAT
+Sidwell-2021-100-25	25	Fish	49	29	227	FISH
+Sidwell-2021-100-26	26	Fly(v.)		30	1441	FLY (MOVE THROUGH AIR)
+Sidwell-2021-100-27	27	Foot	56	31	1301	FOOT
+Sidwell-2021-100-28	28	Full(container)		32	1429	FULL
+Sidwell-2021-100-29	29	Give	60	33	1447	GIVE
+Sidwell-2021-100-30	30	Good	61	34	1035	GOOD
+Sidwell-2021-100-31	31	Green		36	1425	GREEN
+Sidwell-2021-100-32	32	Hair(of head)	65	37	2648	HAIR (HEAD)
+Sidwell-2021-100-33	33	Hand	65	38	1277	HAND
+Sidwell-2021-100-34	34	Head		39	1256	HEAD
+Sidwell-2021-100-35	35	Hear	69	40	1408	HEAR
+Sidwell-2021-100-36	36	Hot	179	43	1286	HOT
+Sidwell-2021-100-37	37	I	78	44	1209	I
+Sidwell-2021-100-38	38	Kill(v.)	82	45	1417	KILL
+Sidwell-2021-100-39	39	Knee		46	1371	KNEE
+Sidwell-2021-100-40	40	Know	83	47	3626	KNOW
+Sidwell-2021-100-41	41	Leaf		48	628	LEAF
+Sidwell-2021-100-42	42	Long(length)		51	1203	LONG
+Sidwell-2021-100-43	43	Louse		52	1392	LOUSE
+Sidwell-2021-100-44	44	Man/Male/husband		53	1554	MAN
+Sidwell-2021-100-45	45	Woman/Female/wife			962	WOMAN
+Sidwell-2021-100-46	46	Many	95	54	1198	MANY
+Sidwell-2021-100-47	47	Moon	96b	55	1313	MOON
+Sidwell-2021-100-48	48	Mouth		57	674	MOUTH
+Sidwell-2021-100-49	49	Name	100	58	1405	NAME
+Sidwell-2021-100-50	50	Neck	103	59	1333	NECK
+Sidwell-2021-100-51	51	New		60	1231	NEW
+Sidwell-2021-100-52	52	Night	105	61	1233	NIGHT
+Sidwell-2021-100-53	53	Nose	106	62	1221	NOSE
+Sidwell-2021-100-54	54	one	109	64	1493	ONE
+Sidwell-2021-100-55	55	Path/Road	120	65	2457	PATH OR ROAD
+Sidwell-2021-100-56	56	Rain	115	67	658	RAIN (PRECIPITATION)
```

Comment:
RAIN could be a noun or a verb, so please use the Concept Set "RAINING OR RAIN" instead.